checkdeps.py

```diff
@@ -10,8 +11,14 @@
     EXTRAS_REQUIRE. This is fine because most developers do, too.
 """
 
+from __future__ import print_function
```

Comment:
Hmm, obviously you meant `from __future__ import absolute_import` rather than `from __future__ import print_function`. Which I'm sure I saw earlier here. Maybe rebasing made git diff confused. Anyway, absolute_imports removed (not needed if we don't support Python 2.6 as you say), while the print_function remains (since you've said you're interested in starting to think about Python3).

Reply:
Same true for print_function:
https://docs.python.org/2/library/__future__.html

Reply:
Keeping this prevents regressions in style and enforces the newer syntax: https://stackoverflow.com/questions/28694380/pylint-says-unnecessary-parens-after-r-keyword

Reply:
but you keep removing the parens...

Reply:
If there was only one or two instances of parens but lots of statements I removed the statements rather than adding the import and enforcing the whole file. If there were mostly parens in a file I changed the print statements to functions and enforced this at the top to avoid regression. I wasn't about to enforce print functions throughout the project. If there's any project-wide tidying up around print I would prefer to change print statements to use logging or sys.stdout.write() as appropriate. That was outside the scope of the task I was undertaking, however.